Amazon.IonObjectMapper.Test/Utils.cs

```diff
@@ -115,15 +115,13 @@ public static T DeserializeWithCustomSerializer<T>(IonSerializer<T> customSerial
             Dictionary<Type, IIonSerializer> ionSerializers, T item)
         {
             var serializer = new IonSerializer(new IonSerializationOptions { IonSerializers = ionSerializers });
-
```

Comment:
Can you revert these line space changes since nothing was done to this file?

Reply:
Reverted.